crypto/cmac/cmac.c

```diff
@@ -139,7 +145,7 @@ int CMAC_Init(CMAC_CTX *ctx, const void *key, size_t keylen,
             return 0;
         if ((bl = EVP_CIPHER_CTX_block_size(ctx->cctx)) < 0)
             return 0;
-        if (!EVP_Cipher(ctx->cctx, ctx->tbl, zero_iv, bl))
+        if (EVP_Cipher(ctx->cctx, ctx->tbl, zero_iv, bl) <= 0)
```

Comment:
I ask because I wondered why the test case did not exhibit the same behavior on 1.1.1
since this EVP_Cipher call returned 0, but it seems that may have changed while
reworking everything at once.  Now I see it is possible to return -1 when an error happens.